invesalius/gui/widgets/canvas_renderer.py

```diff
@@ -690,7 +804,7 @@ def draw_text_box(self, text, pos, font=None, txt_colour=(255, 255, 255), bg_col
 
         return px, py, cw, ch
 
-    def draw_arc(self, center, p0, p1, line_colour=(255, 0, 0, 128), width=2):
+    def draw_arc(self, center, p0, p1, line_colour=(255, 0, 0, 128), width=2) -> None:
```

Comment:
center is tuple of two integers
p0 is tuple of two integers
p1 is tuple of two integers
line_colour is tuple of 4 integers
width is integer

Reply:
I think that center, p0, and p1 should all be a `Tuple[float, float]`